bin/README.md

```diff
@@ -0,0 +1,29 @@
+# funclist
+
+usage:
+```
+./funclist api.json
+```
+
+Look in a OpenApi json file, the list of Functions.
+To do so, look at which object in components.schemas end up with Request.
+Then each functions call are separate by a space
+
+# line_check
+
+usage:
+```
+./line_check "test_keywork" "this_arg" <<< "there is this_arg in this line"
+```
+
+Will return "this_arg".
+
+This utilise was made so we don't have to call grep for each keywork we need to look at.
+With this utility we can just give all our keyword to line_check, and then switch on which keywork was found.
```

Comment:
This utility was created to avoid calling grep for each keyword we need to check. With this utility, we can pass all our keywords to line_check and then switch based on which keyword was found.